_topic_map.yml

```diff
@@ -2464,6 +2464,8 @@ Topics:
     File: customizing-installation-ossm
   - Name: Performance and scalability
     File: ossm-performance-scalability
+  - Name: Troubleshooting your service Mesh
```

Comment:
I think the troubleshooting section should be further down, right before Removing OSSM.

Reply:
I changed the title to "Troubleshooting installation" and kept it where it is, because I think that's where it belongs.

Reply:
From my perspective, troubleshooting comes after deploying apps, security, traffic management, extensions and 3scale. I think the strongest title for this section is just `Troubleshooting` as I look at the content in this assembly. 

What is your thought process about why it belongs here?

Reply:
My thought process is that if a customer cannot get service mesh up and running, if their installation fails, they're not going to be able to do any of the things in security, traffic management, or setting up extensions.

Reply:
@neal-timpe  per our conversation, I've moved the Troubleshooting to just before Removing OSSM.